learn/getting-started/writing-a-restful-api-with-ballerina.md

```diff
@@ -24,11 +24,13 @@ To complete this tutorial, you need:
 
 ## Designing the Two Endpoints 
 
-The first endpoint is about getting data from the service as well as adding data to the service. Therefore, the service should handle both HTTP GET and POST requests. The GET request is to get data whereas the POST request is to add data. In the case of the GET request, the response should be 200 OK whereas in the case of POST request the response should be 201 created.
+The first endpoint is about getting data from the service as well as adding data to the service. Therefore, the service should handle both HTTP GET and POST requests.
+- The GET request is to get data, and the response should be `200 OK`.
+- The POST request is to add data, and the response should be `201 created`.
 
-The second endpoint is about getting filtered data from the service. The data is filtered by the ISO code. Therefore, the second service accepts the ISO code as part of the URL and then responds with 200 OK status code. In the event of an error, the relevant error is sent back to the client.
+The second endpoint is about getting filtered data from the service. The data is filtered by the ISO code. Therefore, the second service accepts the ISO code as part of the URL and responds with `200 OK` status code. In the event of an error, the relevant error is sent back to the client.
 
-Following is the URL for each endpoint respectively.
+Following is the URL for each endpoint, respectively.
```

Comment:
```suggestion
Following is the URL for each endpoint respectively.
```

Reply:
Reasoning for this pls. Grammerly outpu :

<img width="1317" alt="image" src="https://user-images.githubusercontent.com/16300038/151653028-5e833e4c-21de-4e90-96da-600d447a5527.png">